utils/prometheus-exporter/files/metrics/lqm.lua

```diff
@@ -47,7 +47,6 @@ if f then
         "distance",
         "exposed",
         "hidden",
-        "last_tx",
```

Comment:
Why did you remove this one? Both last_tx and last_tx_total are metrics and are different.

Reply:
My mistake, I thought it was redundant with last_tx total.